ctapipe/reco/hillas_reconstructor.py

```diff
@@ -282,7 +285,7 @@ class are set to np.nan
             az_uncert=err_est_dir,
             h_max=h_max,
         )
-
+        result.prefix = self.__class__.__name__
```

Comment:
is there a reason to set this here and not inside the container instantiation (two lines above)? Same in hillas_intersection

Reply:
Just that I did it before introducing the prefix setting in `__init__` feature